docs/guide/cookbook/module.md

```diff
@@ -419,7 +419,50 @@ Routes to be added must be _Array_ type even if it only has one element as you c
 
 ![route_liked_borderline](../images/route_liked.png)
 
-### 2-4. Recipe D (Use hooks)
+### 2-4. Recipe D (Create hooks)
+As you know we have two main kinds of hooks `listener` and `mutator`. In this section, we look into how to create our own ones.
+
+1. Create the `hooks.ts` inside `./core/modules/example-module`.
+
+2. Import `hooks` from `core`:
+
+```js
+import { createListenerHook, createMutatorHook } from '@vue-storefront/core/lib/hooks'
+```
+
+3. Prepare your own `hooks`:
+
+```ts
+const {
+  hook: beforeExampleHook,
+  executor: beforeExampleExecutor
+} = createMutatorHook<any, any>()
+
+const {
+  hook: afterExampleHook,
+  executor: afterExampleExecutor
+} = createListenerHook()
+
+const exampleHooks = {
+  beforeExample: beforeExampleHook,
+  afterExample: afterExampleHook
+}
+const exampleHooksExecutors = {
+  beforeExample: beforeExampleExecutor,
+  afterExample: afterExampleExecutor
+}
+```
+
+4. Export created `hooks`:
+
+```js
+export {
```

Comment:
You should implement `exampleHooksExecutors` in certain place. Because exactly in this place - hooks will be called then. Without that, it does nothing.